package.json

```diff
@@ -47,6 +48,8 @@
     "del": "6.0.0",
     "eslint": "^8.57.0",
     "eslint-config-fbjs": "4.0.0",
+    "mock-fs": "^5.2.0",
+    "json-schema-to-typescript": "^13.0.0",
```

Comment:
Do we actually use this to generate typescript types, or just for json-schema runtime validation?

Reply:
using in a gulp task in the build to automatically generate from the schema file so we don't need to manually keep the two in sync